### PR TITLE
add instructions for applying koji ansible config (SOFTWARE-5341)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: docs/release/new-release-series.md [hunk 1]
```diff
@@ -31,6 +31,20 @@ Prepare Koji and OSG-Build
 
     -   Apply the Koji Ansible config on the Koji Hub host.
 
+            :::console
```
Comment (matyasselmeci) [inline docs/release/new-release-series.md:34]: `--check` is 'dry-run mode' meaning it lets you preview changes without making them; you should add something about running that command again without `--check`.

Comment (edquist) [reply docs/release/new-release-series.md:34]: ok how about now?

Comment (matyasselmeci) [reply docs/release/new-release-series.md:34]: Yep, looks good.

**Comment by edquist** [comment]
link check is holding this up...

**Comment by matyasselmeci** [comment]
I think this is the "error":
> WARNING -  Documentation file 'policy/software-release.md' contains a link to 'release/cut-data-release.md' which is not found in the documentation files.

which is unrelated to this change.

**Pull request merged**